scripts/styles.sh

```diff
@@ -6,14 +6,25 @@ OUTPUT_STYLECSS=internal/view/assets/css/style.css
 INPUT_ARCHIVECSS=internal/view/assets/less/archive.less
 OUTPUT_ARCHIVECSS=internal/view/assets/css/archive.css
 
+# Detect support of avx2
+
+if [ -x "$(command -v bun)" ]; then
+    if grep -q avx2 /proc/cpuinfo; then
+        BUNEXCUTE="bun"
```

Comment:
Maybe just call the variable `BUN`, the same way we don't write `CC_EXECUTE` but just `CC` to refer generically to the local C compiler.